purchase_order_hide_receipt_status/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Order Hide Receipt Status",
+    "version": "16.0.1.0.0",
+    "category": "Purchase",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "license": "AGPL-3",
+    "author": "Quartile Limited, Odoo Community Association (OCA)",
+    "depends": ["purchase_stock", "purchase_reception_status"],
```

Comment:
`purchase_reception_status` is not a technical dependency, so let's remove it.

```suggestion
    "depends": ["purchase_stock"],
```